README.md

```diff
@@ -366,3 +360,7 @@ ISAR_MQTT_PASSWORD
 ```
 
 If not specified the password will default to an empty string.
+
+# Contributions
+
+Equinor welcomes all kinds of contributions, including code, bug reports, issues, feature requests, and documentation. Please initiate your contribution by creating an [issue](https://github.com/equinor/isar/issues) or by forking the project and making a pull requests. Commit messages shall be written according to [this guide](https://cbea.ms/git-commit/).
```

Comment:
I prefer keeping the line length shorter for readability, f.eks around 120 characters

Reply:
Well spotted! I did not notice it was all in one line :)